packages/react-examples/.storybook/fluentuiTheme.js

```diff
@@ -0,0 +1,33 @@
+import { create } from '@storybook/theming';
+import logo from '../public/Fluent.svg';
+
+// Theming and branding the storybook to fluent. Taken from https://storybook.js.org/docs/react/configure/theming
+export default create({
+  base: 'light',
+
+  // Storybook-specific color palette
+  colorPrimary: 'rgba(255, 255, 255, .4)',
+  colorSecondary: '#0078d4',
```

Comment:
does it make sense to grab these color values from one of the themes (maybe `webLightTheme`) from `@fluentui/react-theme` ?

Reply:
can you pls create follow up issue on this? ty

Reply:
sure issue #18009